pontoon/db/__init__.py

```diff
@@ -30,6 +30,9 @@ def process_lhs(self, qn, connection):
         lhs, params = super(IContainsCollate, self).process_lhs(qn, connection)
         if self.collation:
             lhs = lhs.replace('::text', '::text COLLATE "{}"'.format(self.collation))
+            if '::text' not in lhs:
+                lhs = lhs.replace(
+                    '."string"', '."string"::text COLLATE "{}"'.format(self.collation))
```

Comment:
I'll look into that issue today, because I don't have enough context to decide if We should leave this hard-coded field.

Reply:
@jotes its used only once in the whole code i.e. in /base/models.py for translation 'string' query. So if its going to be used only there or further only for string query in the future its safe i guess

Reply:
Yes, I'm aware where it's used in Pontoon. However, I don't know what has been changed in Django 1.11

Reply:
@jotes for whatever reason the query no longer had `::text` so the collation mangling failed

the latest update to this PR seems to be good i think